test/compress/global_defs.js

```diff
@@ -123,6 +126,8 @@ mixed: {
         console.log(CONFIG);
     }
     expect: {
+        const FOO = { BAR: 0 };
+        console.log(FOO.BAR);
```

Comment:
Interesting side effect. That wasn't my intention. Even if `FOO` was declared as `const FOO = null;` I was thinking that since `FOO` is global then `FOO.BAR` is fair game to be replaced with the `global_defs` setting.